docs/userGuide/tweakingThePageStructure.md

```diff
@@ -306,4 +306,63 @@ In this case `myHead.md` will override the `chapterLayout/head.md`.
 
 **The layout named `default` (if any) is automatically applied to every single page.** When you `init` a MarkBind site, MarkBind also generates an empty `default` layout.
 
+## Toggling alternative contents in a page
+
+You can use tags to selectively filter HTML elements when building a site.
+
+Tags are specified by the `tags` attribute, **and can be attached to any HTML element**.
+
+<div class="indented">
+
+{{ icon_example }} Attaching tags to elements:
+```html
+# Print 'Hello world'
+
+<p tags="language--java">System.out.println("Hello world");</p>
+<div tags="language--C#">Console.WriteLine("Hello world");</div>
+<span tags="language--python">print("Hello world")</span>
```

Comment:
Is there a reason why we use different HTML tags for different languages: `<p>`, `<div>` and `<span>`? It doesn't come across naturally in terms of what an author would do, since there's no reason to put code snippets of different languages in different types of HTML tags.

Reply:
It's supposed to demonstrate that the `tags` attribute can be on different types of elements, but I get how that can be a little confusing, will change